graph/collection-properties.md

```diff
@@ -0,0 +1,402 @@
+
+```HTTP
+PATCH /interestingData
+{
+  "foos": [
+    {
+      "someProperty": "a replacement value"
+    },
+    {
+      "someProperty": "more replacement value"
+    },
+    {
+      "someProperty": "a new value demonstrating that additional elements can be in the collection"
+    }
+  ]
+}
+
+204 No Content
+```
+
+#### Check the new contents of the collection
+
+```HTTP
+GET /interestingData/foos
+
+200 OK
+{
+  "value": [
+    {
+      "someProperty": "a replacement value"
+    },
+    {
+      "someProperty": "more replacement value"
+    },
+    {
+      "someProperty": "a new value demonstrating that additional elements can be in the collection"
+    }
+  ]
+}
+```
+
+### Entity Type
+
+#### Replace the elements in a collection
+
+```HTTP
+PATCH /interestingData
+{
+  "bars": [
+    {
+      "id": "fourthBarId",
+      "differentProperty": 20
+    },
+    {
+      "id": "fifthBarId",
+      "differentProperty": -99999
+    }
+  ]
+}
+
+204 No Content
+```
+
+#### Check the new contents of the collection
+
+```HTTP
+GET /interestingData/bars
+
+200 OK
+{
+  "value": [
+    {
+      "id": "fourthBarId",
+      "differentProperty": 20
+    },
+    {
+      "id": "fifthBarId",
+      "differentProperty": -99999
+    }
+  ]
+}
+```
+
+#### Change the elements in a collection
+
+```HTTP
+PATCH /interestingData/bars
+{
+  "@context": "#$delta",
+  "value": [
+    {
+      "id": "fifthBarId",
+      "differentProperty": 1024
+    },
+    {
+      "id": "sixthBarId",
+      "differentProperty": 9801
+    },
+    {
+      "id": "fourthBarId",
+      "@removed": {}
+    }
+  ]
+}
+
+204 No Content
+```
+
+#### Check the new contents of the collection
+
+```HTTP
+GET /interestingData/bars
+
+200 OK
+{
+  "value": [
+    {
+      "id": "fifthBarId",
+      "differentProperty": 1024
+    },
+    {
+      "id": "sixthBarId",
+      "differentProperty": 9801
+    }
+  ]
+}
+```
+
+## Exceptions
+
+It might not make sense for a collection property to be a collection of entities in cases where the collection must be viewed holistically; that is, cases where one element in the collection impacts or is coupled with another element in the collection. 
+The most common example of this would be a list of "tasks" that should be executed in order.
+Inserting a single task into a collection only makes sense because the client knows the preceeding and proceeding tasks.
+In cases like this, it might make sense for a collection to use a complex type instead of an entity type.
+However, even in this case, it might make sense to use a collection of entity types anyway so that clients can individually address the tasks in the list, having URLs to individual tasks that can be referred to. 
+In such a case, the workload should simply not support a delta `PATCH` request on this collection.
+
+## Conclusion
+
+Collections of entity types have several behaviors that are not available for collections of complex types:
+1. Individual elements can be retrieved
```

Comment:
Note that the corollary of this is that items in the collection can be referenced by other navigation properties